pkgs/by-name/fo/fosrl-olm/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+# versionCheckHook,
```

Comment:
Should remove the version check instead of leaving commented out. It’s trivial to readd at a later date.